### ISSUE TITLE
readme.md docker container doesn't exist.

### ISSUE THREAD

**Issue by CHBMB** [OmerTu/GoogleHomeKodi#199]
Hey @keydon 

Just noticed the `README.md` had been updated and I changed the docker container I'm running to reflect this, however [omertu/googlehomekodi](https://hub.docker.com/r/omertu/googlehomekodi) doesn't actually have a container associated with it.  

Thought about submitting a PR to change it back to the previous `sinedied/googlehomekodi` container but wasn't sure if the intention is to trigger a build on the `omertu` docker hub repository.

Great work on this project by the way, got it all setup, pretty straightforward, through a Nginx reverse proxy with SSL, althought damn, I wish IFTTT had a less laborious way of adding applets. :grinning:

**Comment by keydon** [comment]
thats true omertu/googlehomekodi contains currently no actual docker images. I failed creating a docker-repo for multiple architectures like (amd46 [regualar pc], armv6 [raspi 1], armv7 [raspi 2], armv8 [raspi 3]). Tried it with different crosscompiling/claud/virtualization services. maybe in the future...

But I also removed `docker pull` from the `README.md` replacing it with a `docker build` command.
The advantage of this approch is, that thanks to the fact that the base node docker image is availalbe in all those mentioned and many more architectures. And since it does not take too long building that image yourself, I decided to go this way, since I value beeing able to run this app on any hardware more than the comfort of a prebuilt image.
